applications/virtual-fly-brain/client/src/components/ErrorDialog.js

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment } from 'react';
+import Button from '@mui/material/Button';
+import { MailIcon, BugIcon, ModalErrorIcon } from '../icons';
+import Modal from '../shared/modal/Modal';
+
+const ErrorDialog = ({ display, message }) => {
+    const [open, setOpen] = React.useState(display);
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    React.useEffect(() => {
+        setOpen(display);
+    }, [display]);
+
+    return (
+        <Modal
+            open={true}
```

Comment:
@Aiga115 Hi Aigul, I think this got in by mistake, can you please change this to use the state property 'open'?

Reply:
@jrmartin I see, sorry, will fix that